Automattic/jetpack
# Hide menu items for p2 sites

Fixes 1287-gh-Automattic/p2 and 2385-gh-Automattic/p2

#### Changes proposed in this Pull Request:

* Hide menu items that are not relevant for P2 sites
* Hide Settings / Performance for all P2 sites, fixes 1287-gh-Automattic/p2-
* Hide Jetpack section for free p2 sites 2385-gh-Automattic/p2
* Hide some more settings sections for Hubs that are not making any sense like Writting, Reading, Discussion

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply diff generated by this PR
* Check Calypso menus for p2, hubs and regular sites are updated to reflect the above requirements
